messaginginapp/src/test/java/io/customer/messaginginapp/ui/controller/InlineMessageViewControllerTest.kt

```diff
@@ -0,0 +1,128 @@
+package io.customer.messaginginapp.ui.controller
+
+import io.customer.commontest.config.TestConfig
+import io.customer.commontest.config.testConfigurationDefault
+import io.customer.commontest.extensions.assertCalledOnce
+import io.customer.commontest.extensions.assertNoInteractions
+import io.customer.commontest.extensions.random
+import io.customer.messaginginapp.state.InAppMessagingAction
+import io.customer.messaginginapp.state.InAppMessagingManager
+import io.customer.messaginginapp.testutils.core.JUnitTest
+import io.customer.messaginginapp.testutils.extension.createInAppMessage
+import io.customer.messaginginapp.ui.bridge.EngineWebViewDelegate
+import io.customer.messaginginapp.ui.bridge.InAppHostViewDelegate
+import io.customer.messaginginapp.ui.bridge.InAppPlatformDelegate
+import io.mockk.clearMocks
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.spyk
+import org.amshove.kluent.shouldBeEqualTo
+import org.amshove.kluent.shouldBeNull
+import org.junit.jupiter.api.Test
+
+class InlineMessageViewControllerTest : JUnitTest() {
```

Comment:
should we also test size changes updates? as in `animateViewSize` is being called with correct params?

Reply:
It is covered in behavior tests in #552